requirements-devel.txt

```
@@ -25,7 +25,7 @@ codespell==2.3.0
 colorama==0.4.6
 coverage==7.6.1
 craft-application==4.2.7
-craft-archives==2.0.0
+craft-archives @ git+https://github.com/canonical/craft-archives@work/CRAFT-3538-noble-key-conflict
```

Just putting this here to make it easy if you take my suggestion about releasing today.

```suggestion
craft-archives==2.0.1
```